academy/3-ibc/3-channels.md

```diff
@@ -21,30 +21,37 @@ In this section, you will:
 
 Connections and clients comprise the main components of the transport layer in IBC. However, application to application communication in IBC is conducted over **channels**, which route between an application module such as the module which handles Interchain Standard (ICS) 20 token transfers on one chain, and the corresponding application module on another one. These applications are namespaced by **port identifiers** such as 'transfer' for ICS-20 token transfers.
 
-<HighlightBox type="info">
+<HighlightBox type="note">
 
-Note that, in the case of interchain accounts, there are two different port IDs for host and controller modules:
+Note that it's possible to have either a symmetric case, where the application logic is independent of the direction of the packet being sent, as well as an asymmetric case where packets should only be sent in one direction or the logic can be different depending on the send direction. It makes sense (but is not imposed) to use the same port ID on both application modules in the symmetric case, and different port ID on either chain in the asymmetric.
 <br/><br/>
+For example, in the case of interchain accounts, there are two different port IDs for host and controller modules:
 `icahost` is the default port id that the interchain accounts host submodule binds to, whereas `icacontroller-` is the default port prefix that the interchain accounts controller submodule binds to.
```

Comment:
@tmsdkeys Can you clarify two language queries:

1. Should the two instances of "the interchain accounts" in this line use a possessive form? Either: 

- **the interchain account's**, meaning "belonging to one account"; or
- **the interchain accounts'**, meaning "belonging to multiple accounts" (in which case the uses of "submodule" should become plural)

2. Is `icacontroller-` correct, or is the trailing "-" a typo?

Reply:
Hey Andrew, 

1. So in fact we refer to the 'interchain accounts' module, i.e. a module that allows for the creation and authentication of interchain accounts. But I see the confusion which we may solve by saying "the interchain accounts module's host/controller submodule. Potentially even putting i_nterchain accounts_ in italics if that matches the style rules.
2. This is correct, but could benefit from having some dots added `icacontroller-...`

Reply:
Many thanks!

Reply:
Alternatively, perhaps clarifying in the previous line works better. A new version could read (temp emphasis to highlight changes):

For example, in the case of **the** interchain accounts **module**, there are two different port IDs for **the** host and controller **submodules**: `icahost` is the default port ID that the interchain accounts host submodule binds to, whereas `icacontroller-...` is the default port prefix that the interchain accounts controller submodule binds to.

Reply:
Not sure about using 'the interchain accounts module' in the first instance, as there I understand it more as 'interchain accounts, the feature' rather than 'interchain accounts, the module'...

Reply:
I see. So a few alternatives: 

1. For example, in the case of interchain accounts, there are two different port IDs for the host and controller submodules: `icahost` is the default port ID that the **interchain accounts module's** host submodule binds to, whereas `icacontroller-...` is the default port prefix that the controller submodule binds to.

2. For example, in the case of interchain accounts, there are two different port IDs for the host and controller submodules: **in the interchain accounts module**, `icahost` is the default port ID that the host submodule binds to, whereas `icacontroller-...` is the default port prefix that the controller submodule binds to.

I think the second option is easier to read, as it separates the modules/submodules elements (and removes the need to use the possessive form "accounts module's", which is a little harder to parse).

Reply:
2!